src/node_modules/app/machines/builder/_context.js

```diff
@@ -12,8 +15,10 @@ export function createBuilderStores () {
 		selectedRequestTab: writable('fields'),
 		showFullResponse: writable(false),
 		// doc strings
-		activeDocs: writable(''),
-		aggDocText: writable(''),
+		activeDocs: writable(DEFAULT_FIELD_DOCS),
+		aggDocText: writable(DEFAULT_AGG_DOCS),
+		defaultFieldDocs: DEFAULT_FIELD_DOCS,
+		defaultAggDocs: DEFAULT_AGG_DOCS,
```

Comment:
I'd rather not have constants in an objects that's supposed to host stores.
I'd just import `DEFAULT_FIELD_DOCS` and `DEFAULT_AGG_DOCS` in `docs.options.js` as suggested earlier.